sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -448,8 +449,10 @@ public partial class CommunityGalleryImageVersionData : Azure.ResourceManager.Co
     {
         internal CommunityGalleryImageVersionData() { }
         public System.DateTimeOffset? EndOfLifeOn { get { throw null; } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public bool? ExcludeFromLatest { get { throw null; } }
         public Azure.Core.ResourceIdentifier Id { get { throw null; } }
+        public bool? IsExcludedFromLatest { get { throw null; } }
```

Comment:
Maybe we don't need to rename this property, in which case we wouldn't need to add the customized code？

Reply:
well i changed a few of this case to align our current standard of polishing the SDK